plugins/slack_webhooks/test/decorators/stage_decorator_test.rb

```diff
@@ -27,4 +27,32 @@
       stage.slack_webhooks.size.must_equal 0
     end
   end
+
+  describe "#slack_buddy_channels" do
+    it "returns channels when for_buddy is true" do
+      stage.slack_webhooks = [SlackWebhook.new(channel: "test", webhook_url: "http://slack.com/abcdef",
```

Comment:
foo(
  bar: 1,
  foo: 2
)

On Thu, Jun 23, 2016 at 7:49 PM, Zehua Liu <notifications@github.com> wrote:

> In plugins/slack_webhooks/test/decorators/stage_decorator_test.rb
> <https://github.com/zendesk/samson/pull/1058#discussion_r68347356>:
>
> > @@ -27,4 +27,32 @@
> >        stage.slack_webhooks.size.must_equal 0
> >      end
> >    end
> > +
> > +  describe "#slack_buddy_channels" do
> > +    it "returns channels when for_buddy is true" do
> > +      stage.slack_webhooks = [SlackWebhook.new(channel: "test", webhook_url: "http://slack.com/abcdef",
>
> Getting complaints from rubocop when I move the next line to 2-space
> indent instead of aligning with the hash key in the previous line:
>
> plugins/slack_webhooks/test/decorators/stage_decorator_test.rb:34:9: C: Align the elements of a hash literal if they span more than one line.
>         stage_id: stage.id, for_buddy: true)]
>         ^^^^^^^^^^^^^^^^^^
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1058/files/1c0e3b4444e4a02023ed261f0c6443f9203994e5#r68347356>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAAsZ3OaCsR36m4q22eVTw5k5lvKptXoks5qO0WygaJpZM4Iyxjs>
> .
>